### ISSUE TITLE
Feature add: Antilogs

### ISSUE THREAD

**Issue by mrblackcity** [Qalculate/qalculate-gtk#476]
Having antilogs would be nice to have.

**Comment by rickhg12hs** [comment]
`10^x`, `2^x` , and `e^x` are available functions.
![image](https://user-images.githubusercontent.com/6000188/235003309-4bb4e35d-de13-49d1-a881-10715e0df268.png)

**Comment by hanna-kn** [comment]
I do not understand how a specific antilog function would be useful. There are already multiple ways to raise a base by a power (I can't see how the antilog would differ from exponentiation).

```
solve(log10(x)=3) = exp10(3) = pow(10, 3) = 10^3 = 1E3 = 1000

solve(log(x, b)=n) = pow(b, n) = b^n
```

You can easily create your own antilog function (e.g. using the expression `x^y`).

**Comment by cousteaulecommandant** [comment]
Indeed, isn't an antilogarithm pretty much the same as a power (maybe swapping the two arguments, i.e. `alog(x, b) := pow(b, x)`)?  Do you think there would be added value in providing an `alog()` function name as an alias for `pow()`?
